cli/src/commands/new.rs

```diff
@@ -41,7 +41,7 @@ use crate::ui::Ui;
 /// For more information, see
 /// https://github.com/martinvonz/jj/blob/main/docs/working-copy.md.
 #[derive(clap::Args, Clone, Debug)]
-#[command(group(ArgGroup::new("order").args(&["insert_after", "insert_before"])))]
+#[command(group(ArgGroup::new("target").args(&["revisions", "insert_after", "insert_before"])))]
```

Comment:
nit: these arguments conceptually belong to the same group, but this `ArgGroup` is noop. We'll unblock `--insert-before`/`after` combination, and the `revisions` has default value.